evennia/utils/verb_conjugation/pronouns.py

```diff
@@ -407,15 +407,8 @@ def pronoun_to_viewpoints(
 
     # step down into the mapping
     viewpoint_map = PRONOUN_MAPPING[viewpoint]
-    
-    if pronoun_type in viewpoint_map:
-        pronouns = viewpoint_map[pronoun_type]
-    else:
-        pronouns = viewpoint_map[DEFAULT_PRONOUN_TYPE]
-    if gender in pronouns:
-        mapped_pronoun = pronouns[gender]
-    else:
-        mapped_pronoun = pronouns[DEFAULT_GENDER]
+    pronouns = viewpoint_map.get(pronoun_type) or viewpoint_map.get(DEFAULT_PRONOUN_TYPE)
+    mapped_pronoun = pronouns.get(gender) or pronouns.get(DEFAULT_GENDER)
```

Comment:
Sorry for giving the wrong suggested fix before, but the way to write this would be 
```python
pronouns = viewpoint_map.get(pronoun_type, viewpoint_map[DEFAULT_PRONOUN_TYPE])
mapped_pronoun = pronouns.get(gender, pronouns[DEFAULT_GENDER])
```

Reply:
Oh! Okay, I'll change that.

For my own information, what makes the syntax `mapped_pronoun = pronouns.get(gender, pronouns[DEFAULT_GENDER])` better than `mapped_pronoun = pronouns.get(gender) or pronouns[DEFAULT_GENDER]` ?

Reply:
Both work. Generally it's considered more 'Pythonic' to make use of the in-built Python primitives to their fullest. So since `dict.get(value, default)` is a thing, one should make sure to use that functionality explicitly rather than mimic the functionality by separately checking if `get(value)` returns `None` and then use the default value as a 'home brew' solution.